plugin.js

```diff
@@ -5,27 +5,80 @@ var fs = require("fs");
 var path = require("path");
 var AdmZip = require("adm-zip");
 var events = require("events");
+var rmdir = require("rimraf"); // I still don't believe this is really a thing
+var pluginParser = require('./plugin-parser.js');
 
 var fileName = null;
+var localState = null;
```

Comment:
But in `bindDeleteButtons()` technically speaking `localState` is not needed since `state` is what is needed. Instead it's needed in `installPlugin()` which has no direct access to this variable.

So, basically this `localState` is a global variable in `configure.js`. I understand that the lifecycle of this variable is the same as the one of the whole program, so there is no issue in saving an extra reference to it in `localState`.

However, I'd be happier with an accessor in `configure.js` allowing to access this variable when it is not directly available in the scope of the caller (otherwise it can passed as argument as for `bindDeleteButtons`.